go.mod

```diff
@@ -48,3 +48,8 @@ require (
 	k8s.io/klog/v2 v2.9.0 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.1.2 // indirect
 )
+
+replace (
+	golang.org/x/net => golang.org/x/net v0.0.0-20220906165146-f3363e06e74c
+	golang.org/x/text => golang.org/x/text v0.3.8
```

Comment:
why not pin to 0.3.8 in the `go.mod`?

Reply:
@reasonerjt Sometimes the replace is necessary due to embedded/indirect dependencies -- indirect deps might call for specific (older) versions. I haven't verified that it's needed here. The probelm with replace CVE fixes is that we tend to forget to remove them later when we really need newer versions  than the replace, which means it ends up downgrading instead of upgrading the dep.

Reply:
I follow a previous Azure CVE fix PR to use the `replace` directive instead of `require`, and I think maybe using `replace` for indirect refereced module is more intuitive, although `require` directive should also work here.

I agree. We should at least go through the module file per release. The replace or require would grow out of date by time, then seperating them from the `require` section into `replace` is more helpful to distinguish.